core/conversation-manager/index.js

```diff
@@ -42,7 +42,7 @@ const actions = {
                 let recipientId = session.uid;
 
                 if (recipientId) {
-                    let msg = "Think of me as your personal shopping assistant.";
+                    let msg = "Hi! Think of me as your personal shopping assistant.";
```

Comment:
We shouldn't say Hi! in the help message because this gets sent for EVERY help message... I think down where we handle the get started button press we should just add a send text message call with any greeting stuff instead of in the help message.